packages/path_provider/path_provider_linux/pubspec.yaml

```diff
@@ -11,17 +11,17 @@ flutter:
         pluginClass: none
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.12.0-0 <3.0.0"
   flutter: ">=1.10.0"
 
 dependencies:
-  path: ^1.6.4
+  path: ^1.8.0-nullsafety.3
   xdg_directories: ^0.1.0
```

Comment:
It looks like this dependency isn't migrated yet. You might want a send a PR to https://github.com/flutter/packages/tree/master/packages/xdg_directories. cc @gspencergoog

Reply:
I have already landed a PR https://github.com/flutter/packages/pull/250 here to migrate `xdg_directories` to nnbd.